standards_lab/api/urls.py

```diff
@@ -12,4 +12,14 @@
         views.ProjectUploadFile.as_view(),
         name="project-upload",
     ),
+    path(
+        "project/<slug:name>/process_start",
```

Comment:
Are we following any APIs guideline? I would suggest the url like `project/<slug:name>/process/start`

Reply:
Yeah ideally just one endpoint: ` /project/<slug:name>/process`

```
GET => { process(s) status ... }
POST <= { start: "cove" } (or something, I can foresee other processes in the future)
```
That said it may be easier to merge and then change this later.

Reply:
When things are left for later, sometimes later never comes. In case this is going to be merged like it is, then at least could you change the "_" for "-"?

Reply:
I've updated the endpoint with these suggestions https://github.com/OpenDataServices/standards-lab/pull/19/commits/39625fabad06c7c8d0b655745cc2b97cd878a3be

Reply:
Thanks @Bjwebb